eng/pipelines/templates/jobs/smoke-tests.yml

```diff
@@ -0,0 +1,171 @@
+parameters:
+  - name: Daily
+    type: boolean
+    default: true
+  - name: ServiceDirectory
+    type: string
+    default: ""
+  - name: Artifacts
+    type: object
+    default: []
+
+jobs:
+  - ${{ if and(eq(parameters.Daily, false), ne(variables['Skip.Release'], 'true')) }}:
+    - job: smoke_test_eligibility
+      variables:
+        - template: /eng/pipelines/templates/variables/globals.yml
+      pool:
+        vmImage: $(OSVmImage)
+      displayName: Check Smoke Test Eligibility
+      steps:
+        - template: /eng/pipelines/templates/steps/use-node-version.yml
+          parameters:
```

Comment:
You should be able to remove the `parameters:` section.  The `use-node-version.yml` template will use `$(NodeVersion)` by default.

Reply:
Ah good catch, I still had it in there from when it was using NodeTestVersion